tailwind.config.js

```diff
@@ -37,6 +38,7 @@ module.exports = {
       },
       fontSize: {
         'base': '0.9375rem',
+        'content': "15px"
```

Comment:
`0.9375rem` are `15px`. So you can skip `content` here. Also, `base` will be applied globally as default to all elements.